docs/specs/2018-04-03.md

```diff
@@ -786,4 +786,14 @@ Example:
 Filter: 
   Domain: ['www.taobao.com','www.tmall.com']
   
-```
\ No newline at end of file
+```
+
+#### Default Role
+
+The default role will be generated when the policy is specified or when the service is configured with attributes such as vpc and nas that explicitly require specific permissions.
```

Comment:
The default role will be generated when policies are specified or when the service is configured with properties such as VpcConfig or NasConfig, which explicitly require some permissions.